pkg/apis/core/validation/validation.go

```diff
@@ -1108,6 +1112,14 @@ func validatePathNoBacksteps(targetPath string, fldPath *field.Path) field.Error
 	return allErrs
 }
 
+// validateNonFlagArgument verifies that the argument cannot be used as a flag parameter.
+func validateNonFlagArgument(arg string, fldPath *field.Path) field.ErrorList {
```

Comment:
probably validate at point of use in the volume plugin as well, since data can already exist in the system?

Reply:
done